Sources/SwiftProtobuf/PathDecoder.swift

```diff
@@ -35,7 +35,9 @@ extension Message {
     guard let type = Self.self as? any _ProtoNameProviding.Type else {
       return nil
     }
-    return type._protobuf_nameMap.number(forJSONName: field)
+    return Array(field.utf8).withUnsafeBytes { bytes in
```

Comment:
Isn't this trying to return an Array now instead of an Int?

Reply:
The return type of `withUnsafeBytes` is the same as its closure return type and in the closure we return an Int.

[withUnsafeBytes documentation](https://developer.apple.com/documentation/swift/array/withunsafebytes(_:))

Reply:
Sorry, should have gotten my morning coffee before looking at the change. 😄